### ISSUE TITLE
Gate: unable to edit_order price & amount at the same time after upgrade

### ISSUE THREAD

**Issue by Tongoko** [ccxt/ccxt#17549]
### Operating System

Windows

### Programming Languages

Python

### CCXT Version

3.0.62

### Description

I upgraded today the ccxt version from 2.0.68 to 3.0.62. Before the upgrade, my bot was able to edit order price and amount at the same time. After the upgrade however, I'm receiving the error "Only can choose one of amount or price".
I then downgraded ccxt to 2.0.68 and now it's working again.
It seems that somewhere between the upgrades, the option to edit price and amount at the same time got lost. 
Can this be brought back again?

### Code

```
exchange.create_limit_buy_order(symbol, amount, price, params={'timeInForce': 'PO'})

exchange.editOrder(order_id, symbol, 'limit', 'buy', amount, price, params={'timeInForce': 'PO'})

```

**Comment by carlosmiei** [comment]
Hello @Tongoko, thanks for reporting it we will try to fix this issue asap.

**Comment by frosty00** [comment]
@Tongoko we only support the v4 api for gateio, the v2 api is deprecated and may be removed by them in the future

**Comment by Tongoko** [comment]
@frosty00 ccxt 2.0.68 is already using v4, so I don't think that is the problem.
![1681785257944677990026120795359](https://user-images.githubusercontent.com/117158948/232655569-57f8b470-d789-4da2-846d-0e760b98660d.jpg)


@carlosmiei Could you maybe show me which code I can manually edit with the old version so the editOrder functions the same as the old version?

Thanks for the replies so far!

**Comment by Tongoko** [comment]
Update:
It seems that on version 2.0.68, gate.py didn't have editOrder in the code yet and it was added in a later version.

I went to the async and normal version of gate.py and deleted all lines containing edit.
Now it's working again to edit price and amount at the same time.

My guess is that in the background, it's not editing but canceling the order and creating a new one, which serves my purpose for now.
